packages/abcm2ps/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+# Contributor: Tuyen Pham <tuyenpm9@gmail.com>
+# Contributor: Lex Black <autumn-wind@web.de>
+# Contributor: Simon Thorpe <simon@hivetechnology.com.au>
+# Contributor: kess <joe@macmahon.eu>
+# Contributor: damir <damir@archlinux.org>
+# Contributor: Luis Pablo Gasparotto <lpgasparotto(at)gmail(dot)com>
+
+pkgname=abcm2ps
+pkgver=8.14.13
+pkgrel=2
+pkgdesc='Convert ABC music notation files to PostScript from the commandline'
+arch=(x86_64 aarch64)
+url='http://moinejf.free.fr/'
+license=(GPL3)
+depends=(pango)
+makedepends=(python-docutils)
+conflicts=(abcm2ps-devel)
+groups=(abc pro-audio)
```

Comment:
The "abc" group was in the AUR `PKGBUILD`. I can't remember whether I added it. Should I leave it in? (The idea being, that there will be other ABC-related software, like `abc2svg`, `abcmidi` etc., in the same group.)

Reply:
Sure, good idea! Probably a few other plugin bundles split up their source in separate projects with separate releases, so it's good practice to group them.

Reply:
Ok, left it in.